.rubocop.yml

```diff
@@ -1,12 +1,18 @@
 require:
   - rubocop-rails
+  - rubocop-rspec
+  - rubocop-performance
 
 AllCops:
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 3.0
```

Comment:
It should be Ruby 2.6, which is our minimum supported version.

Reply:
fixed that. Thx!